src/templates/homepage.html

```diff
@@ -8,6 +9,29 @@
   <p class="desktop-promo-separator-text">{{ _('More to Explore') }}</p>
 {% endif %}
 
+{% defer (url=anonApi('search')|urlparams(limit=11, doc_type='website'), pluck='objects', id='feed-websites') %}
+  {% if this %}
+    <div class="feed feed-items feed-home c" data-columns>
+    {% if this.length >= 3 %}
+      {{ website_tabs(this|sliceArray(0, 3)) }}
+    {% endif %}
+    {% if this.length >= 6 %}
```

Comment:
No, we want to show the first one if there are at least 3, and then the second one if there are at least 3 more.